iceoryx_examples/complexdata/topic_data.hpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef IOX_EXAMPLES_COMPLEXDATA_TOPIC_DATA_HPP
+#define IOX_EXAMPLES_COMPLEXDATA_TOPIC_DATA_HPP
+
+#include "iceoryx_utils/cxx/forward_list.hpp"
+#include "iceoryx_utils/cxx/list.hpp"
+#include "iceoryx_utils/cxx/optional.hpp"
+#include "iceoryx_utils/cxx/stack.hpp"
+#include "iceoryx_utils/cxx/string.hpp"
+#include "iceoryx_utils/cxx/variant.hpp"
+#include "iceoryx_utils/cxx/vector.hpp"
+
+using namespace iox::cxx;
```

Comment:
ok here in this small example, but usually not a good idea in head files.

Reply:
I decided to use the namespace for better readability, especially in the (not yet finished) readme.md

Reply:
Sure, as I said. Would this be a library header I would reject this argument.